src/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl.py

```diff
@@ -670,20 +670,22 @@ def __call__(
                 [Common Diffusion Noise Schedules and Sample Steps are Flawed](https://arxiv.org/pdf/2305.08891.pdf).
                 Guidance rescale factor should fix overexposure when using zero terminal SNR.
             original_size (`Tuple[int]`, *optional*, defaults to (1024, 1024)):
-                TODO
+                Part of SDXL's micro-conditioning as explained in section 2.2 of [https://huggingface.co/papers/2307.01952](https://huggingface.co/papers/2307.01952).
+                If `original_size` is not the same as `target_size` the image will appear to be down- or upsampled. `original_size` defaults to `(width, height)` if not specified.
```

Comment:
What do you think about shortening it to like the below so it's a bit shorter/easier to read (big ➕ for linking to hf.co/papers!), and I think it'd be clearer to lead with what the parameter does?

```suggestion
                If `original_size` is not the same as `target_size` the image will appear to be down- or upsampled. `original_size` defaults to `(width, height)` if not specified (see [SDXL micro-conditioning](https://huggingface.co/papers/2307.01952) for more details).
```

Reply:
Swapped the two sentences - would like to keep the "section 2.2" part so that people don't have to search the whole paper. Let me know what you think :-)